### PR TITLE
Level 5a Scene.

### PR DESCRIPTION
**Pull request by jayfriso**
The scene for level 5a. Tested and playable. Here is the doc to see how it looks like. https://docs.google.com/presentation/d/1Hs6NHdKqHP752UBvTKt042_GYW9RC-k0RYtG33wmPC4/edit#slide=id.p.

Along with the level 5a scene here are the chagned files.
- Spinning Blade prefab and script = Prefab was not applied. Script automatically set the rotation direction at start so I changed this.
- Scene 4b - just changed the rotating blade to have a rotation of 90 (do to script change).
- LockedDoor script - There was no way to make it so the door is initially open and then closes on trigger so I changed it so this is settable.

image

### CODE REVIEW

File: Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs [hunk 1]
```diff
@@ -9,24 +9,44 @@ public class receiveSignalLockedDoor : receiveSignal {
 	private Collider2D collider;
 
     public switchScene doorToNextLevel;
+    public string opensOrClosesUponTrigger = "opens";
```
Comment (jeaniewu) [inline Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs]: is there a reason why it has to be string instead of boolean?

Comment (jayfriso) [inline Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs]: Yeah Ill change it, couldn't come up with a good variable name for a boolean lol

File: Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs [hunk 2]
```diff
@@ -9,27 +9,27 @@ public class receiveSignalLockedDoor : receiveSignal {
...
-        if (opensOrClosesUponTrigger == "closes") {
+        if (opensUponTrigger == false) {
             openDoor();
         }
     }
 
 
     public override void activate() {
-       if (opensOrClosesUponTrigger == "opens") {
+       if (opensUponTrigger == true) {
```
Comment (jeaniewu) [inline Assets/Scripts/receiveSignal/receiveSignalLockedDoor.cs]: lol jay just do if(opensUponTrigger) and if(!opensUponTrigger)

**Comment by jeaniewu** [comment]
@jayfriso after u fix that u can merge 👍

**Pull request merged**